### PR TITLE
build: add std_srvs as missing dependency

### PR DESCRIPTION
**Pull request by Minipada**
See title.
Used here: 

https://github.com/alireza-hosseini/ipcamera_driver/blob/118fffba1674c8d5f1090aa7656452ebf87a4cd5/src/ipcamera_driver.cpp#L111

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -3,6 +3,7 @@ project(ipcamera_driver)
 
 
 find_package(catkin REQUIRED COMPONENTS
+  std_srvs
```
Comment (alireza-hosseini) [inline CMakeLists.txt]: List of packages should be sorted alphabetically.

File: CMakeLists.txt [hunk 2]
```diff
@@ -15,6 +16,7 @@ find_package( OpenCV REQUIRED )
 include_directories(include ${catkin_INCLUDE_DIRS}  ${OpenCV_INCLUDE_DIRS} )
 catkin_package(
  CATKIN_DEPENDS
+  std_srvs
```
Comment (alireza-hosseini) [inline CMakeLists.txt]: same as above.

**Review by alireza-hosseini — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (Minipada) [reply CMakeLists.txt]: done

File: CMakeLists.txt (hunk 2 above)
Comment (Minipada) [reply CMakeLists.txt]: done

**Comment by alireza-hosseini** [comment]
Thanks for your PR.

**Review by alireza-hosseini — APPROVED** [review]

**Pull request merged**